Core/include/Acts/EventData/TrackStateProxy.hpp

```diff
@@ -667,9 +676,41 @@ class TrackStateProxy {
   /// @note This is mainly to copy explicitly a projector from one state
   ///       to another. If you have a projection matrix, set it with
   ///       `setProjector`.
+  /// @deprecated Use setProjector(span) instead
+  //[[deprecated("use setProjector(span) instead")]]
   void setProjectorBitset(ProjectorBitset proj) requires(!ReadOnly) {
+    BoundMatrix projMatrix = bitsetToMatrix<BoundMatrix>(proj);
+    FullProjectorMapping fullMapping =
+        projectorToIndices<eBoundSize>(projMatrix);
+    setFullProjectorMapping(fullMapping);
+  }
+
+  FullProjectorMapping fullProjectorMapping() const {
+    assert(has<hashString("projector")>());
+    return component<FullProjectorMapping, hashString("projector")>();
+  }
+
+  template <std::size_t measdim>
+  ProjectorMapping<measdim> projectorMapping() const {
```

Comment:
I guess eventually we'd want to cycle these back around to just be `projector`? Otherwise we could call them `measurementMappingFunction` or `calibratedMappingFunction` since that's what they're called in the KF literature.